docs/src/main/paradox/external-connectors.md

```diff
@@ -25,4 +25,8 @@ TODO: Add links to more connectors. [Contributions](https://github.com/akka/alpa
 
 [Streaming TCP Java documentation](http://doc.akka.io/docs/akka/2.4/java/stream/stream-io.html#Streaming_TCP)
 
-[Streaming TCP Scala documentation](http://doc.akka.io/docs/akka/2.4/scala/stream/stream-io.html#Streaming_TCP)
\ No newline at end of file
+[Streaming TCP Scala documentation](http://doc.akka.io/docs/akka/2.4/scala/stream/stream-io.html#Streaming_TCP)
+
+## Azure
```

Comment:
specifically it is a connector to Azure IoT Hub underlying storage. Do you think the section title should be more specific ? My reasoning about using a generic "Azure" category is to group future connectors (e.g. Azure tables, Azure service bus, etc) under the same umbrella. At the same time we could also differentiate though, and have Azure Storage and Azure IoT. I'm open to suggestions.